README.md

```diff
@@ -1,4 +1,31 @@
-# codewind-odo-extension
-Extension to Codewind providing support for odo projects: https://codewind.dev
+# Codewind ODO extension
+Extension to Codewind providing support for ODO projects: https://codewind.dev
 
-README.md is in progress and will be completed soon.
+## Add additional rules to support Codewind ODO extension
+The ODO extension needs to add additional rules for accessing OpenShift resources
```

Comment:
Add a colon (:) or period (.) at the end of the sentence. I'm thinking colon because this sentence precedes and sets up the following steps.